### ISSUE TITLE
isNull type in DayjsUtils was incompatible with IUtils

### ISSUE THREAD

**Issue by cssguy39** [dmtrKovalenko/date-io#598]
Fixed type for `isNull` to allow for `null`. Was getting this error:

```
../node_modules/@date-io/dayjs/build/dayjs-utils.d.ts:27:5 - error TS2416: Property 'isNull' in type 'DayjsUtils<TDate>' is not assignable to the same property in base type 'IUtils<TDate>'.
  Type '(date: Dayjs) => boolean' is not assignable to type '(value: TDate | null) => boolean'.
    Types of parameters 'date' and 'value' are incompatible.
      Type 'TDate | null' is not assignable to type 'Dayjs'.
        Type 'null' is not assignable to type 'Dayjs'.

27     isNull: (date: Dayjs) => boolean;
       ~~~~~~
```

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/@date-io/dayjs/build/dayjs-utils.d.ts b/node_modules/@date-io/dayjs/build/dayjs-utils.d.ts
index 093bbe5..bd2c26a 100644
--- a/node_modules/@date-io/dayjs/build/dayjs-utils.d.ts
+++ b/node_modules/@date-io/dayjs/build/dayjs-utils.d.ts
@@ -24,7 +24,7 @@ export default class DayjsUtils<TDate extends Dayjs = Dayjs> implements IUtils<T
     date: (value?: any) => TDate;
     toJsDate: (value: Dayjs) => Date;
     isValid: (value: any) => boolean;
-    isNull: (date: Dayjs) => boolean;
+    isNull: (date: Dayjs | null) => boolean;
     getDiff: (date: TDate, comparing: TDate, units?: Unit) => number;
     isAfter: (date: TDate, value: TDate) => boolean;
     isBefore: (date: TDate, value: TDate) => boolean;
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by dmtrKovalenko** [comment]
Yep this make sense, should be updated to `TDate | null`

**Comment by dmtrKovalenko** [comment]
Released in 2.13.0
